profile/README.md

```diff
@@ -0,0 +1,29 @@
+# Welcome to Nebari! 👋
+
+<!-- TODO: Verify and update all the links -->
+
+<a href="https://nebari.dev">
+    <img src="nebari-banner.svg" alt="A banner showing the Nebari logo witt the tagline 'A data science platform: Built for scale, designed for collaboration.' This is followed by a get started button pointing to Nebari 101"/>
+</a>
+
+---
+
+📖 Looking to learn about Nebari? Check out [our documentation](https://www.nebari.dev/).
+
+🪴 Want to report a bug or have a new feature request? [Open an issue](https://github.com/nebari-dev/nebari/issues/new/choose).
+
+💬 Have a question about using Nebari? Ask on [our community forum](https://github.com/orgs/nebari-dev/discussions).
+
+💎 Wish to contribute to Nebari? Check out [our contributing guidelines](https://www.nebari.dev/community/#how-to-contribute).
+
+✨ Curious what we're working on? Check out [our roadmap](https://github.com/nebari-dev/governance).
+
+---
+
+<!-- We can pin the following repositories, in order:
```

Comment:
Maybe for now `nebari` `nebari-docs` only?

Reply:
Sounds good!